core/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -111,6 +112,13 @@ public boolean allowAliasesToMultipleIndices() {
         return (id & FORBID_ALIASES_TO_MULTIPLE_INDICES) == 0;
     }
 
+    /**
+     * @return whether aliases should be ignored (when resolving a wildcard)
+     */
+    public boolean ignoreAliases() {
+        return (id & IGNORE_ALIASES) != 0;
+    }
+    
     public void writeIndicesOptions(StreamOutput out) throws IOException {
         out.write(id);
```

Comment:
this method and the read need to handle backwards compatibility: see https://github.com/elastic/elasticsearch/commit/7548b2edb782a2732aca5e9bae9016c6a01cb6e6 for a similar change with bw comp layer from when we added `allowAliasesToMultipleIndices`